### PR TITLE
Wrote in ability to pause/play time

### PR DESCRIPTION
**Pull request by ojustino**
This consists of a couple of new functions in `core.py` and corresponding entries in `wwt_json_api.js`.

### CODE REVIEW

**Comment by astrofrog** [comment]
@ojustino - can you rebase this pull request? (to pick up the latest fixes to tests)

**Comment by ojustino** [comment]
Done.

File: pywwt/core.py [hunk 1]
```diff
@@ -157,6 +158,39 @@ def resume_tour(self):
         """
         self._send_msg(event='resume_tour')
 
+    def pause_time(self):
+        """
+        Pause the progression of time in the viewer.
+        """
+        if self._paused == False:
```
Comment (astrofrog) [inline pywwt/core.py]: Change this to ``if not self._paused:``

File: pywwt/core.py [hunk 2]
```diff
@@ -157,6 +158,39 @@ def resume_tour(self):
...
+            self._send_msg(event='pause_time', state=self._paused)
+            self._paused = not self._paused
+
+    def play_time(self):
+        """
+        Resume the progression of time in the viewer.
+        """
+        if self._paused == True:
```
Comment (astrofrog) [inline pywwt/core.py]: ``if self._paused:``

**Pull request merged**